src/scenes/_Router/GalleryRoute.tsx

```diff
@@ -86,7 +96,21 @@ export default function GalleryRoute({
     }
   }, [footer, footerVisibleOutOfView, footerVisibleWithinView, footerIsFixed, isMobile]);
 
-  const banner = useMemo(() => <Banner text="" queryRef={query} />, [query]);
+  const banner = useMemo(() => {
+    return isFeatureEnabled(FeatureFlag.POSTER_PAGE) ? (
+      <Banner
+        title={countdownTimer}
+        text="Thank you for being a member of Gallery. Celebrate our new brand with us by signing our 2022 Community Poster that we will mint as an NFT."
+        queryRef={query}
+        localStorageKey="gallery-poster-banner"
+        requireAuth
+      >
+        <NavLink to="/object006">Sign Poster</NavLink>
+      </Banner>
```

Comment:
i don't think the "Sign Poster" should necessarily be passed as `children`, since it describes the action component instead of the main text! how about something like this:

```js
      <Banner
        title={countdownTimer}
        text="Thank you for being a member of Gallery. Celebrate our new brand with us by signing our 2022 Community Poster that we will mint as an NFT."
        queryRef={query}
        localStorageKey="gallery-poster-banner"
        requireAuth
        actionComponent={<NavLink to="/object006">Sign Poster</NavLink>}
      />
```

then in `Banner.tsx` you can just render `{actionComponent}`

Reply:
Beautiful 😍. Thanks for the suggestion robin!